README.md

```diff
@@ -1,24 +1,220 @@
+- Git
+- Node
+- Ruby
+- Rails
+- Postgres
+- Postman
+
+### Setup
+Clone this repository to your desired folder:
+
+```sh
+  git clone git@github.com:Diegogagan2587/Book-a-concert-back-end.git
+  cd Book-a-concert-back-end
+```
+
+### Install
+
+Install the project dependencies with:
+
+```sh
+  bundle install
+  rails db:drop
+  rails db:create
+  rails db:schema:dump
+  rails db:migrate
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh 
+  rails server
+```
+
+### Run tests
+
+To run tests, run the following command:
+```sh
+  rspec
+```
+- Open your web browser and visit http://127.0.0.1:3000
+- You should now see the main view of Rails page.
+- You can interact with the backend API using Postman and following this [documentation](https://documenter.getpostman.com/view/31013872/2s9YXk3Lv4)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## Authors <a name="authors"></a>
+
+**Rudi Carrillo**
+
+- GitHub: [@rudicarrilloypr](https://github.com/rudicarrilloypr)
+- Twitter: [@__rudicarrillo](https://twitter.com/__rudicarrillo)
+- LinkedIn: [Rudi Carrillo](https://www.linkedin.com/in/rudi-carrillo/)
+- Instagram: [@__rudicarrillo](https://www.instagram.com/_rudicarrillo/)
+
+**Dievo Vidal Lopez**
+
+- GitHub: [@Diegogagan2587](https://github.com/Diegogagan2587)
+- Twitter: [@dieg02587](https://twitter.com/dieg02587)
+- LinkedIn: [Diego Vidal Lopez](https://www.linkedin.com/in/diego-vidal2587/)
+
+**Dani Morillo**
+
+- GitHub: [@danifromecuador](https://github.com/danifromecuador)
+- Twitter: [@danifromecuador](https://twitter.com/danimorilloc)
+- LinkedIn: [danielfromecuador](https://www.linkedin.com/in/danimorilloc)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## Acknowledgements <a name="acknowledgements"></a>
+
+I'd like to express my gratitude to everyone who made this project possible.
+
+Thanks again to everyone who believed in this project and helped make it a reality!
+
+Furthermore, special recognition is due to the [author](https://www.behance.net/muratk) of this [design](https://www.behance.net/gallery/26425031/Vespa-Responsive-Redesign) that served as inspiration for our work
+
+
+<!--- Future Features ---->
+
+## <a href="#future-features"></a>Future Features
+- Improve UI design
+- Add profile picture
+- Add Bio to each profile
+- Improve user log in and signup security
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- CONTRIBUTING -->
+
+## Contributing <a name="contributing"></a>
+
+Contributions are welcome! If you find any bugs or want to enhance the project with new features, please follow these steps:
+
+1. Fork the repository.
+2. Create a new branch for your feature or bug fix.
+3. Make the necessary changes and commit them.
+4. Push your changes to your forked repository.
+5. Submit a pull request describing your changes.
+
+Feel free to check the [issues page](../../issues/).
+
```

Comment:
Link to issue page should match the github issue page:
https://github.com/Diegogagan2587/Book-a-concert-back-end/issues

Reply:
Thank you for catching that detail, I fixed that and other broken links